docs/src/main/asciidoc/writing-extensions.adoc

```diff
@@ -487,7 +487,9 @@ Quarkus provides `create-extension` Maven Mojo to initialize your extension proj
 It will try to auto-detect its options:
 
 * from `quarkus` (Quarkus Core) or `quarkus/extensions` directory, it will use the 'Quarkus Core' extension layout and defaults.
+ifndef::no-quarkiverse[]
```

Comment:
this should not be excluded. This is just stating that if you generate using the quarkiverse layout which actually is how users should use it.

Reply:
Sorry, my mistake. I've removed all changes to this file.